core/src/main/java/com/linecorp/armeria/server/Http1RequestDecoder.java

```diff
@@ -280,6 +281,15 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) throws Exception
                     final boolean endOfStream = contentEmpty && !transferEncodingChunked;
                     this.req = req = DecodedHttpRequest.of(endOfStream, eventLoop, id, 1, headers,
                                                            keepAlive, inboundTrafficController, routingCtx);
+                    if (Flags.allowLargeRequestEarlyRejection()) {
```

Comment:
I think it's probably fine to remove the flag and add content-length header check as the default behavior.

1. From the user's standpoint, there is almost no change in behavior
2. The behavior seems reasonable enough to include as opt-in

Reply:
cool, remove the flag!